Public/Src/Cache/DistributedCache.Host/Service/Internal/CacheServerFactory.cs

```diff
@@ -25,7 +25,7 @@ namespace BuildXL.Cache.Host.Service.Internal
     /// <summary>
     /// Creates and configures cache server instances.
     /// </summary>
-    internal class CacheServerFactory
+    public class CacheServerFactory

```

Comment:
Nit: Add comment saying this is used externally.

It wasn't obvious nor expected when I renamed it